tests/e2e/basic/conftest.py

```diff
@@ -91,6 +88,7 @@ def get_equipment_id(instantiate_controller, testbed):
     if len(CONFIGURATION[testbed]['access_point']) == 1:
         equipment_id = instantiate_controller.get_equipment_id(
             serial_number=CONFIGURATION[testbed]['access_point'][0]['serial'])
+    print(equipment_id)
```

Comment:
What is our strategy on print vs logging for the test framework?

Reply:
pytest can generate the log files within its framework
prints will be only printed if we use -vvv as to see more verbosed outputs

eventually i will be moving prints like thses to a log file alligned to the framework not just pytest related